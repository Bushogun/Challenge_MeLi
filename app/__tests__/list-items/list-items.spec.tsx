// import React from 'react';
// import { renderWithProviders } from '@/src/utils/test-utils';
// import { screen } from '@testing-library/dom';
// import userEvent from '@testing-library/user-event';
// import { ListItems } from '@/src/components/list-items/list-product-items';
// import configureStore from 'redux-mock-store';

// jest.mock('react-redux', () => ({
//   useSelector: jest.fn(),
//   useDispatch: jest.fn(),
// }));

// describe('ListItems component', () => {
//   const mockStore = configureStore([]);
//   let store;

//   beforeEach(() => {
//     store = mockStore({
//       product: {
//         itemsPerPage: 10,
//         products: [
//           // Define aquí los productos de prueba según tu estructura
//         ],
//       },
//     });
//   });

//   it('should render correctly', () => {
//     useSelector.mockReturnValue({
//       product: {
//         itemsPerPage: 10,
//         products: [
//           // Define aquí los productos de prueba según tu estructura
//         ],
//       },
//     });

//     const { container } = renderWithProviders(<ListItems />, store);
//     expect(container).toMatchSnapshot();
//   });

//   // Agrega más pruebas según tus necesidades
// });
