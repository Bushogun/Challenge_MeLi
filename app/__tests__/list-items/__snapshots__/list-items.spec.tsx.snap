// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItems component should render correctly with products 1`] = `[Function]`;
